hexrd/ui/main_window.py

```diff
@@ -85,7 +85,8 @@ def __init__(self, parent=None, image_files=None):
         self.load_widget.ui.setVisible(False)
         self.workflow_widgets[WORKFLOW_HEDM].append(self.load_widget.ui)
 
-        self.import_data_widget = ImportDataPanel(self.ui)
+        self.import_data_widget = ImportDataPanel(self.color_map_editor,
```

Comment:
Its not great to have the import panel depend on the color map editor. I wonder if there is another way, may be with signal connected together here? May be the import data widget could emit an 'updating started' and 'complete signal' which could connected by some function here to enable/disable the color map editor. Lets think about it a bit.

Reply:
Ok, we can hold on this for a bit to figure a better implementation.